.github/workflows/ci-a11y-vrt-experimental.yml

```diff
@@ -0,0 +1,117 @@
+name: Accessibility and Visual Regression Experimental Tests
+
+on:
+  push:
+    branches:
+      - main
+      - next
```

Comment:
We've used `next` as the branch name for the next major release. However, I recently created a `beta` branch for automatically publishing prereleases with the `beta` tag.

Should we start using `beta` here or should we make the `next` brach publish beta prereleases?

Reply:
I can update this to `beta`, thanks for the context!

Reply:
I can update the other one as well

Reply:
I kept both :)